toolbox/arcgee/map.py

```diff
@@ -16,6 +16,182 @@
 import ee
 
 
+def list_color_ramp():
+    """
+    List all supported color ramps.
+    """
+    return [
+        "viridis",
+        "magma",
+        "plasma",
+        "cividis",
+        "blues",
+        "greens",
+        "reds",
+        "purples",
+        "orange_red",
+        "spectral",
+        "turbo",
+    ]
+
+
+def get_color_ramp(ramp_name):
+    """
+    Return a list of hex color codes based on the color ramp name.
+
+    Supported ramps:
+        - viridis
+        - magma
+        - plasma
+        - cividis
+        - blues
+        - greens
+        - reds
+        - purples
+        - orange_red
+        - spectral
+        - turbo
+
+    Args:
+        ramp_name (str): Name of the color ramp (case-insensitive).
+
+    Returns:
+        list of str: Hex color codes.
+    """
+    ramps = {
```

Comment:
I would avoid hard coding these color ramps. 1) this is hard to maintain, adds to code bloat, and makes it hard to add new colormaps.

Arcpy Python env has matplotlib built in so I suggest making this dynamic and creating the colormaps hex codes on-the-fly from there. Here is a [good example](https://github.com/gee-community/geemap/blob/master/geemap/colormaps.py#L73).

> Note: As much as I don't like copying code from another package, I don't think we should include geemap as a dependency for this one function because it can make more conflicts on install.